spec/features/admin/delete_custom_organisation_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'support/confirmation_use_case'
+
+describe 'deleting a custom organisation name' do
+  include_examples 'confirmation use case spy'
+  let!(:custom_org_name1) { CustomOrganisationName.create(name: 'DummyOrg1') }
+  let!(:custom_org_name2) { CustomOrganisationName.create(name: 'DummyOrg2') }
+  let!(:custom_org_name3) { CustomOrganisationName.create(name: 'DummyOrg3') }
```

Comment:
This no factory model management is spreading

Reply:
Last time it will be used 👁